README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
```

Comment:
I'm going for "right handed" to indicate counter clockwise rotations. Do
you think clockwise has meaning in R3? Cause that would be simpler.

On 9 May 2016 at 13:10, Chris Foster <notifications@github.com> wrote:

> In README.md
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62446637>:
>
> > @@ -1 +1,160 @@
> > -# Rotations
> > +# Rotations.jl
> > +
> > +This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
> > +
> > +This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
>
> Is the (right handed) in brackets for a reason? Presumably you can have
> the full cross product {active,passive}x{RH,LH}
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62446637>
>